_posts/2017/2017-08-17-tracing-the-suspect.md

```diff
@@ -0,0 +1,245 @@
+---
+layout: post
+title: "Tracing the suspect - a microservices murder mystery"
+date: "2017-08-17 09:30:00"
+image: blog-header/tracing-the-suspect.jpg
```

Comment:
1st: difficult to understand it'S footprints
2nd: mysterious, but not yet convincing
3rd: good footprints visualisation, just missing some kind of crime scene analogy.

Really undecided on that topic, but not too keen on finding the perfect fit.